docs/references/api/cors.rst

```diff
@@ -32,3 +35,16 @@ A ``POST`` preflight request would look like this:
   Access-Control-Allow-Methods: GET, POST, PATCH, PUT, DELETE, OPTIONS, HEAD
   Access-Control-Allow-Headers: Authorization, Content-Type, Accept, Accept-Language, Content-Language
   Access-Control-Max-Age: 86400
+
+.. _allowed_origins:
+
+Allowed Origins
+===============
+
+With the below config setting, PostgREST will accept CORS requests from domains :code:`http://example.com` and :code:`http://example2.com`.
```

Comment:
```suggestion
With the following config setting, PostgREST will accept CORS requests from domains :code:`http://example.com` and :code:`http://example2.com`.
```

TIL that "bellow" [is not commonly used as an adjective](https://english.stackexchange.com/a/611).